### PR TITLE
Publish links to Slovak and Polish language variants

### PR DESCRIPTION
**Pull request by pavelkaly**

### CODE REVIEW

**Comment by JuroUhlar** [comment]
Hey, thanks for the pull request! 

* The domains are movapp.cz, sk.movapp.eu, pl.movapp.eu, the domains are live so pls fix your links to make them work
* The first flag should be Czech-Ukrainian, not Brittish-Ukrainian (the czech version should be in the same figma file somewhere)
* Not sure about moving the App download buttons inside the white rectangle, the previous UX opinion is that the white rectange is for things inside the web, mobile apps do not semantically belong there and should be below. @met What do you think?

**Comment by met** [comment]
@JuroUhlar Yes. I agree will all you said.

**Comment by pavelkaly** [comment]
1.) Great, I will do that, links visible on the website (under the flags) can stay in the format as they are? 
2.) Ok, so there should be 4 flags in this section? (cz, en, sk, pl)
3. ) Yees, I did this version exactly according to Figma file but I think the best approach would be to leave apps buttons under white rectangle and just add a heading above these buttons (so it would have the same design as flags section below)

**Comment by JuroUhlar** [comment]
1) Yes! Format is fine, just adjust the link texts please :)
2) No, there is no english version, I don't know why its in the mockup. It's more of an ideation I guess. Sorry, I should have checked the mock-up in more detail before. We have three language variants: movapp.cz, sk.movapp.eu, pl.movapp.eu
3) Yes, let's do that. Two designers, two opinions, but @met says below, we put them below. Thanks! :)

File: pages/index.tsx [hunk 1]
```diff
@@ -28,6 +31,18 @@ const HEARTS_IMAGE: Record<CountryVariant, ReactNode> = {
   pl: <Image src={HeartsUkraine_PL} alt="Polskie i Ukraińskie flagi w sercach" width={140} height={164} />,
 };
 
+interface Languages {
```
Comment (JuroUhlar) [inline pages/index.tsx:34]: Rename this `CountryVariantFlag` or something specific like that. We already have a type `Language` that is heavily used throughout the app.

**Review by JuroUhlar — CHANGES_REQUESTED** [review]

File: pages/index.tsx (hunk 1 above)
Comment (pavelkaly) [reply pages/index.tsx]: renamed

File: pages/index.tsx [hunk 2]
```diff
@@ -169,9 +184,31 @@ const Home: NextPage = () => {
           </div>
         </div>
       </div>
-      <div className="mx-auto my-12 flex items-center flex-wrap justify-center">
-        <DownloadAppStore />
-        <DownloadGooglePlay />
+      {/* Apps section */}
```
[the remark this answers is not in the document]
Comment (pavelkaly) [reply pages/index.tsx]: both components ready :)

File: pages/index.tsx [hunk 3]
```diff
@@ -28,6 +31,18 @@ const HEARTS_IMAGE: Record<CountryVariant, ReactNode> = {
   pl: <Image src={HeartsUkraine_PL} alt="Polskie i Ukraińskie flagi w sercach" width={140} height={164} />,
 };
 
+interface Languages {
+  svg: ReactNode;
+  link: string;
+  country: string;
```
[the remark this answers is not in the document]
Comment (pavelkaly) [reply pages/index.tsx]: used

File: components/basecomponents/DownloadAppStore.tsx [hunk 4]
```diff
@@ -6,14 +6,19 @@ export const DOWNLOAD_IOS_URL = 'https://apps.apple.com/app/apple-store/id161776
...
+        className="bg-black text-white px-1 py-1 rounded-lg flex border-2 border-gray-400 border-gray min-w-[216px]"
+      >
         <div>
-          <AiFillApple size={56} className="mb-1 mr-1" />
+          <AiFillApple size={49} className="mb-1 mr-1" />
         </div>
         <div className="flex flex-col justify-center mr-1">
-          <div className="text-sm leading-none whitespace-nowrap">{t('homepage.download_on')}</div>
-          <div className="text-3xl leading-none whitespace-nowrap">App Store</div>
+          <div className="text-sm leading-none whitespace-nowrap font-light uppercase">{t('homepage.download_on')}</div>
```
[the remark this answers is not in the document]
Comment (pavelkaly) [reply components/basecomponents/DownloadAppStore.tsx]: I see... I was probably inspired by wrong button on google images :( fixed

File: pages/index.tsx [hunk 5]
```diff
@@ -169,9 +184,31 @@ const Home: NextPage = () => {
...
+            <DownloadGooglePlay />
+          </div>
+        </div>
+      </div>
+      {/* Languages section */}
+      <div className="mx-auto mb-12 mt-8 flex flex-col items-center">
+        <h2 className="">Movapp mluví dalšími jazyky</h2>
+        <div className="flex flex-col sm:flex-row items-center">
+          {LanguagesFlags.filter(({ country }) => country !== getCountryVariant()).map(({ svg, link }, i) => (
+            <Link href={`https://${link}`} passHref={true} key={i}>
```
[the remark this answers is not in the document]
Comment (pavelkaly) [reply pages/index.tsx]: fixed

**Review by JuroUhlar — APPROVED** [review]

**Pull request merged**